web/vtadmin/src/components/routes/tablet/FullStatus.tsx

```diff
@@ -30,8 +44,22 @@ const FullStatus: React.FC<Props> = ({ tablet }) => {
             </div>
         );
     }
+    if (!isNil(data) && !isNil(data.status)) {
```

Comment:
Here you can simplify and do:
```suggestion
    if (data && data.status) {
```

Reply:
Fixed